src/changes-backwards-compatible.apib

```diff
@@ -140,3 +140,6 @@ We list all backwards-compatible additions here. These are currently available i
 
 - We already had a part of the products APIs for our own usage, but since it might be useful to you, we are making it public.
   So `products.list` and `products.info` are now available for usage.
+
+- We added a new sorting option `weighted_value` to `deals.list` and will allow ascending as well as descending orders on sorts
+  The default remains created_at and descending as before if nothing is passed.
```

Comment:
Q: same as above.

B: let's add this in the `Latest` section rather than at the end of `January 2019`.

PP: Separate sentences with a dot. Newlines do not add a visible break.